packages/ChangesReporting/Annotation/AnnotationExtractor.php

```diff
@@ -29,8 +29,8 @@ public function extractAnnotationFromClass(string $className, string $annotation
         $pattern = '#' . preg_quote($annotation, '#') . '\s+(?<content>.*?)$#m';
         $matches = Strings::match($docComment, $pattern);
 
-        if ($matches && $matches['content']) {
-            return trim($matches['content']);
+        if ($matches && isset($matches['content'])) {
```

Comment:
What is in `$matches` exactly? The `Strings::match()` returns array only if named arguments are found. So checking it here again does not make sense.
There must be some bug in regular expression.